### PR TITLE
membership sort by name and show 5

### PR DESCRIPTION
**Pull request by CoryITpro**
This is for fixing https://github.com/Joystream/pioneer/issues/2334.
https://dao.joystream.org/#/bounty/preview/267

### CODE REVIEW

**Comment by freakstatic** [comment]
LGTM!

**Comment by singulart** [comment]
@CoryITpro Please address the failing checks.

File: packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx [hunk 1]
```diff
@@ -34,10 +34,10 @@ export const SwitchMemberModal = () => {
 
   const filteredMembers = useMemo(() => {
     if (modalData?.membersToShow) {
-      return members.filter((member) => modalData.membersToShow?.includes(member.id))
+      return members.filter((member) => modalData.membersToShow?.includes(member.id)).sort((a,b) => (a.handle > b.handle) ? 1 : ((b.handle > a.handle) ? -1 : 0))
```
Comment (thesan) [inline packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx:37]: The main problem with this way is that sorts uppercase char first, so `Z` comes before `a`. There is a `Comparator` function in `packages/ui/src/common/model/Comparator.ts` which should work well here:
```suggestion
      return members.filter((member) => modalData.membersToShow?.includes(member.id)).sort(Comparator(false, 'handle'))
```
(it just uses `string.localCompare(string)` under the hood)

File: packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx [hunk 2]
```diff
@@ -34,10 +34,10 @@ export const SwitchMemberModal = () => {
...
     }
 
-    return members
+    return members.sort((a,b) => (a.handle > b.handle) ? 1 : ((b.handle > a.handle) ? -1 : 0))
```
Comment (thesan) [inline packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx:40]: Same here:
```suggestion
    return members.sort(Comparator(false, 'handle'))
```

**Review by thesan — CHANGES_REQUESTED** [review]
Hello @CoryITpro, thank you for your work. Please address the changes requested below.

**Comment by CoryITpro** [comment]
Completed

**Comment by singulart** [comment]
Hi! @CoryITpro Can I ask why is this PR closed? From what I see in the comments, there were some improvements suggested by the reviewer. Have you had a chance to address them properly? In any case, you shouldn't have closed this PR before all comments are addressed..

**Comment by dmtrjsg** [comment]
@CoryITpro get well soon buddy, let us know when you are back so we can resume work on this one..

**Pull request closed without merging**